test/colony-task-work-rating.js

```diff
@@ -137,13 +148,16 @@ contract("Colony Task Work Rating", () => {
     it("should allow revealing a rating by evaluator and worker", async () => {
       await fundColonyWithTokens(colony, token, INITIAL_FUNDING);
       const taskId = await setupRatedTask({ colonyNetwork, colony, token });
+
       const roleManager = await colony.getTaskRole.call(taskId, MANAGER_ROLE);
-      assert.isTrue(roleManager[1]);
```

Comment:
This wasn't worth an `assert.isFalse`, but the equivalent for `roleWorker` was? 😛

Reply:
Oop, that should actually refer to `roleManager[3]`. The difference is that since the manager doesn't submit a rating, we don't need to check their `rateFail` boolean. We need to check the `roleWorker`'s because they submit a rating to the manager. But in any case this test is accessing the wrong field.